src/Frost.test.ts

```diff
@@ -0,0 +1,110 @@
+import * as nock from 'nock'
+import { describe } from 'riteway'
+import { Frost } from './Frost'
+import { Path, Network } from './utils/utils'
+
+describe('Frost createApiToken()', async (should: any) => {
+  const { assert } = should('')
+  const host = 'https://api.frost.po.et'
+
+  const config = {
+    email: 'test@test.com',
+    password: 'test',
+    timeout: 0,
+    host,
+  }
+
+  {
+    const serverResponse = { apiToken: '1' }
+    const token = 'test'
+
+    nock(host)
+      .post(Path.TOKENS)
+      .reply(200, serverResponse)
+
+    const frost = new Frost(config)
+
+    const actual = await frost.createApiToken(token)
+    const expected = serverResponse
+
+    assert({
+      given: 'a status code of 200',
+      should: 'return the object with apiToken equal 1',
+      actual,
+      expected,
+    })
+  }
+
+  {
+    const serverResponse = { apiToken: '1' }
+    const token = 'test'
+    let requestBody = {}
+
+    nock(host)
+      .post(Path.TOKENS, (body: {}) => {
+        requestBody = body
+        return body
+      })
+      .reply(200, serverResponse)
+
+    const frost = new Frost(config)
+    await frost.createApiToken(token)
+
+    const actual = requestBody
+    const expected = {}
+
+    assert({
+      given: 'a request without network',
```

Comment:
If there is no network, wouldn't the nock return something other than 200?

Reply:
sorry, is the property `network` I mean